client/src/utils/tone/index.ts

```diff
@@ -62,20 +62,16 @@ type ToneStates = keyof typeof toneUrls;
 export async function playTone(state: ToneStates): Promise<void> {
   const playSound = !!store.get('fcc-sound');
   if (playSound && toneUrls[state]) {
-    const tone = await import('tone');
+    const Tone = await import('tone');
 
-    if (tone.context.state !== 'running') {
-      tone.context.resume().catch(err => {
-        console.error('Error resuming audio context', err);
-      });
-    }
```

Comment:
Why is this removed? We need to check that the audiocontext state is running, and if not, resume it.

Reply:
reference [doc](https://tonejs.github.io/docs/14.7.77/Context#resume)
I'm using `player.start()` to play audio which is basically same thing. Start audio context from suspended state